docs/List.md

```diff
@@ -109,6 +110,106 @@ export const PostList = (props) => (
 );
 ```
 
+### Bulk Actions
+
+You can replace the list of default bulk actions by your own element using the `bulkActions` prop:
+
+```jsx
+import Button from 'material-ui/Button';
+import { BulkActions, BulkDeleteMenuItem } from 'react-admin';
+import CustomBulkMenuItem from './CustomBulkMenuItem';
+
+const PostBulkActions = props => (
+    <BulkActions {...props}>
+        <CustomBulkMenuItem />
+        {/* Add the default bulk delete action */}
+        <BulkDeleteMenuItem />
+    </BulkActions>
+);
+
+export const PostList = (props) => (
+    <List {...props} bulkActions={<PostBulkActions />}>
+        ...
+    </List>
+);
+```
+
+A custom bulk action, is a material-ui [`MenuItem`](http://www.material-ui.com/#/components/menu), for which you are entirely responsible. It will receive several props allowing it to perform its work:
```

Comment:
I wonder whether we should provide a `BulkAction` component hiding the need to call `onCloseMenu` by the user. Thoughts ?

Reply:
-1 for that, connecting an action creator isn't the end of the world.

Reply:
It's not an action creator, just a kind of callback prop to ensure the bulk action menu is closed after triggering the bulk action